containers/tools/generate-certificates.yml

```diff
@@ -0,0 +1,11 @@
+---
+- name: 'Generate certificates'
+  hosts: all
+  vars:
+    certificates_services:
+      - name: pulp
+      - name: pulp-client
+        common_name: admin
+    content_server_hostname: "content-server-8080-foreman.{{ hostvars[inventory_hostname]['ansible_default_ipv4']['address'] }}.xip.io"
```

Comment:
Doesn't openshift provide some internal hostnames for services?

Reply:
Yes - but this is an external route needed for clients to access today. I think this can be done better in the future but this is just re-organizing what i had already.